### ISSUE TITLE
Link the "xyz Active installs" metric on the sidebar of project pages to the usage stats of the respective project.

### ISSUE THREAD

**Issue by klonos** [backdrop-ops/backdropcms.org#481]
This was brought up on Gitter by @quinnanya:

> ...is https://backdropcms.org/project/usage linked anywhere on the site?

We could link `https://backdropcms.org/project/usage/[projectname]` from the number of active install metric in the sidebar:

<img width="1203" alt="screen shot 2018-11-14 at 8 39 03 am" src="https://user-images.githubusercontent.com/2423362/48444715-d9f5e580-e7e8-11e8-8a96-1159d9ed35e4.png">

**Comment by quinnanya** [comment]
Yes! That would be great. It's really helpful when trying to weigh whether to install a particular module.
